packages/fastify-multer/package.json

```diff
@@ -28,8 +28,8 @@
     "fastify-multer": "2.0.3"
   },
   "peerDependencies": {
-    "@nestjs/common": "^9.0.0 || ^10.0.0",
-    "@nestjs/platform-fastify": "^9.0.0 || ^10.0.0",
+    "@nestjs/common": "^9.0.0 || ^10.0.0 || ^10.30.0",
+    "@nestjs/platform-fastify": "^9.0.0 || ^10.0.0 || ^10.30.0",
```

Comment:
This would already be acceptable due to `^10.0.0`, setting it to `^10.30.0` would make it to where `10.29.x` and lower can no longer be used

Reply:
The problem I ran into was ultimately due to the fastify version and compatibility with reflect-metadata

Reply:
But it is a good idea to keep the packages updated to avoid security issues